pipelines/deduplication/acteur_deduplicate.py

```diff
@@ -0,0 +1,223 @@
+"""
+Script pour dédupliquer les acteurs:
+
+Paramètres requis:
+ - DB_URL_{ENVIRONNEMENT}: connexion base de données
+ - CLUSTERING_CSV_FILEPATH: fichier CSV d'entrée
+    avec 1 ligne = cluster_id -> identifiant_unique
+
+Paramètres optionnelles:
+ - CLUSTER_IDS_TO_CHANGES: pour tester un cluster en particulier
+ - DRY_RUN: pour tester sans modifier la DB
+
+Execution:
+ - python acteur_deduplicate.py
+
+Nommage des variables:
+ - suffixe "_src" pour la donnée de source
+ - suffixe "_db" pour la donnée en base de données
+"""
+
+import json
+import os
+from pathlib import Path
+from typing import List, Tuple
+
+import pandas as pd
+from models.change import Change
+from rich import print
+from rich.prompt import Confirm, Prompt
+from rich.traceback import install
```

Comment:
Je vois qu'on utilise beaucoup [rich](https://rich.readthedocs.io/en/stable/introduction.html) (que je connaissais pas intéressant). 

Il nous vient de `apache-airflow` 

Il serait intéressant de l'ajouter dans nos dev-requirements.in 

Dans `dev-requirements.txt`
```
rich==13.8.1 \
    --hash=sha256:1760a3c0848469b97b558fc61c85233e3dafb69c7a071b4d60c38099d3cd4c06 \
    --hash=sha256:8260cda28e3db6bf04d2d1ef4dbc03ba80a824c88b0e7668a0f23126a424844a
    # via
    #   apache-airflow
    #   flask-limiter
    #   rich-argparse
```